evap/evaluation/tests/test_models.py

```diff
@@ -66,16 +66,16 @@ def test_in_evaluation_to_published(self):
 
     @override_settings(EVALUATION_END_WARNING_PERIOD=24)
     def test_evaluation_ends_soon(self):
-        course = mommy.make(Course, state='in_evaluation', vote_start_datetime=datetime.now() - timedelta(days=2),
-                            vote_end_date=date.today() - timedelta(hours=24), is_graded=False)
+        course = mommy.make(Course, vote_start_datetime=datetime.now() - timedelta(days=2),
+                            vote_end_date=date.today() + timedelta(hours=24))
 
-        Course.update_courses()
+        self.assertFalse(course.evaluation_ends_soon())
+
+        course.vote_end_date = date.today()
         self.assertTrue(course.evaluation_ends_soon())
```

Comment:
Two thoughts:
- This test might possibly fail if executed around midnight and the `date.today()` calls are on another day than the `evaluation_ends_soon()` calls. It's highly improbably that this happens, though, and we don't have an easy way to mock the time (as e.g.  freezegun could do), so I'm okay with leaving it like this. What do you think?
- Since we're overriding the `EVALUATION_END_OFFSET_HOURS` with 0 for this whole test class, we don't really test whether `evaluation_ends_soon` takes that setting into account. Do we want it to?